gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/gateway_query.proto

```diff
@@ -172,6 +178,11 @@ message MetricSeries {
   Period period = 3;
 }
 
+message BaselineMetricSeries {
+  Baseline baseline = 2;
```

Comment:
A metric series doesn't have a baseline outside of the intervals - it's fine to approach it iteratively, but let's leave it empty rather than add a placeholder that wouldn't be used.

Reply:
Currently metric series has only one value later it will be added to interval specific.

Reply:
Even though there will only be one interval, returning it as an interval allows us to evolve the implementation without changing the API (or changing the GQL code at all).